src/MessageHandler/ActivityPub/Inbox/DeleteHandler.php

```diff
@@ -35,16 +40,21 @@ public function __construct(
 
     public function __invoke(DeleteMessage $message): void
     {
-        try {
-            $actor = $this->activityPubManager->findRemoteActor($message->payload['actor']);
-        } catch (\Exception) {
-            return;
-        }
+        $actor = $this->activityPubManager->findActorOrCreate($message->payload['actor']);
+
+        $id = \is_array($message->payload['object']) ? $message->payload['object']['id'] : $message->payload['object'];
+        $object = $this->apActivityRepository->findByObjectId($id);
 
-        if (\is_array($message->payload['object'])) {
-            $object = $this->apActivityRepository->findByObjectId($message->payload['object']['id']);
-        } else {
-            $object = $this->apActivityRepository->findByObjectId($message->payload['object']);
+        if (!$object) {
```

Comment:
add check if $actor is not null/undefined/... thus, something like:

```suggestion
        if (!$object && $actor) {
```

Reply:
So also `findActorOrCreate` can return null/empty objects. Trust me.. >><<

Reply:
$actor is checked by the `instanceof` call in the next if as well, but it won't hurt to check it here as well